stream_alert/rule_processor/threat_intel.py

```diff
@@ -164,14 +164,20 @@ def _extract_ioc_from_record(self, record):
                         for original_key in original_keys:
                             value = value[original_key]
                     if value:
-                        # Threat Intel will only check against public IP addresses
-                        # while processing IP IOCs.
-                        if ioc_type == 'ip' and not self.is_public_ip(value):
+                        # Threat Intel will skip searching for known excluded IOCs
+                        if self.is_excluded_ioc(ioc_type, value):
                             continue
                         ioc_value_type_tuples.add((value, ioc_type))
         return [StreamIoc(value=str(value).lower(), ioc_type=ioc_type, associated_record=record)
                 for value, ioc_type in ioc_value_type_tuples]
 
+    def _setup_excluded_iocs(self, config_excluded_iocs=None):
+        if config_excluded_iocs:
+            self.excluded_iocs = config_excluded_iocs
+        else:
+            self.excluded_iocs = {"ip": set(), "domain": set(), "md5": set()}
+        self.excluded_iocs["ip"] = {IPNetwork(ip) for ip in self.excluded_iocs["ip"]}
```

Comment:
Please stick to single quotes (`'`) vs double quotes (`"`) for string literals to be consistent with the rest of the codebase